scanner/updater/export.go

```diff
@@ -45,32 +45,29 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 	if err != nil {
 		return fmt.Errorf("initializing: manual: %w", err)
 	}
-	bundles["rhel"], err = rhelOpts(ctx)
-	if err != nil {
-		return fmt.Errorf("initializing updater: rhel: %w", err)
-	}
```

Comment:
With this removal should anything under `scanner/updater/rhel/*` be removed as well? perhaps `scanner/updater/rhel/updaterset.go`?

Reply:
yeah I mention in a few comments down I opted to keep that code for now

(1) it limits the size of this PR
(2) helpful should we need to revert if we discover something terrible post-merge

I figure we can keep the old code for some time until we are 100% certain we will not go back